packages/stencil/src/components/datacom-overflow-menu/datacom-overflow-menu-items.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+  Component,
+  h,
+  Prop,
+  Host,
+  Event,
+  EventEmitter,
+  Element,
+} from '@stencil/core';
+import { getSvg } from '../../common/images/icon-provider';
+export type MenuSizeType = 'standard' | 'small';
+
+@Component({
+  tag: 'datacom-menu-items',
```

Comment:
Please change the component name and its tag to singular name as it is referring to a one item only.

Suggestion to 
component: datacom-overflow-menu-item
tag: 'datacom-menu-item'

Reply:
Done